state/machine_test.go

```diff
@@ -2446,6 +2447,27 @@ func (s *MachineSuite) setupTestUpdateMachineSeries(c *gc.C) *state.Machine {
 	return mach
 }
 
+func (s *MachineSuite) addMachineUnit(c *gc.C, mach *state.Machine) *state.Unit {
```

Comment:
We need to add arbitrary units in order to test the machine functionality related to upgrade series.